.gitlab-ci/debian-install.sh

```diff
@@ -54,9 +54,7 @@ apt-get install -y \
 	libpango1.0-dev \
 	libpciaccess-dev \
 	libpixman-1-dev \
-	libselinux1-dev \
 	libspice-protocol-dev \
-	libsystemd-dev \
```

Comment:
Xserver can be (optionally) built with systemd support ? 
Should we build-test it, too ?

Reply:
Also here I'd advocate for a clear separation of concerns, where a display server should not need special treatment for any particular init technology (and especially not one the all-mighty Enterprise Linux company is pushing).

Reply:
In fact, the build failed after I removed this:

> meson.build:96:12: ERROR: Problem encountered: Failed to find an sd-bus provider, tried libsystemd, libelogind, basu.

If we can't even build without a systemd dependency anymore, that would be concerning to me. So imho we should at least ensure that everything works _without_ systemd. (I have never heard of `libelogind` or `basu` but I'd be willing to try them if we can avoid a systemd dependency that way.)

But even more fundamentally, sd-bus seems to be a D-Bus implementation. D-Bus imho yet another Linux-ish overengineered solution a certain Enterprise Linux company is pushing. Imho an X server should not depend on D-Bus _at all_.

Reply:
> In fact, the build failed after I removed `libsystemd-dev` I got:
> 
> > meson.build:96:12: ERROR: Problem encountered: Failed to find an sd-bus provider, tried libsystemd, libelogind, basu.
> 

That really shouldn't happen. It seems the guessing logic in meson.build is wrong:

> if get_option('systemd_logind') == 'auto'
>     build_systemd_logind = build_udev_kms and dbus_dep.found()
> else
>     build_systemd_logind = get_option('systemd_logind') == 'true'
> endif
>

Reply:
> In fact, the build failed after I removed `libsystemd-dev` I got:
> 
> > meson.build:96:12: ERROR: Problem encountered: Failed to find an sd-bus provider, tried libsystemd, libelogind, basu.

https://github.com/X11Libre/xserver/pull/161